### PR TITLE
testing/synergy: new aports

### PR DESCRIPTION
**Pull request by dlintw**

### CODE REVIEW

File: testing/synergy/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+# Contributor: Daniel YC Lin <dlin.tw@gmail.com>
+# Maintainer: Daniel YC Lin <dlin.tw@gmail.com>
+pkgname=synergy
+pkgver=1.8.7
+pkgrel=0
+pkgdesc='Share a single mouse and keyboard between multiple computers'
+url='https://symless.com/synergy/'
+arch="all"
+license="GPL2"
+depends="libxtst libxinerama libxkbcommon avahi curl openssl libxrandr"
```
Comment (ncopa) [inline testing/synergy/APKBUILD]: abuild will handle the dependencies for shared objects. normally `depends` should be empty.

File: testing/synergy/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+prepare() {
+	cd "$builddir"
+	default_prepare || return
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr .. || return 1
+	# unittests don't pass with optimization (segfault on nullptr)
+	sed 's|\-O2|\-O0|g' -i src/test/unittests/CMakeFiles/unittests.dir/flags.make
+	sed 's|\-O2|\-O0|g' -i src/test/unittests/CMakeFiles/unittests.dir/link.txt
+}
```
Comment (ncopa) [inline testing/synergy/APKBUILD]: i would like that the cmake run happens in `build()` not in `prepare()`.

File: testing/synergy/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+"
+builddir="$srcdir/synergy-${pkgver}-stable"
+
+prepare() {
+	cd "$builddir"
+	default_prepare || return
+	mkdir -p build
+	cd build
+	cmake -DCMAKE_INSTALL_PREFIX=/usr .. || return 1
+	# unittests don't pass with optimization (segfault on nullptr)
```
Comment (ncopa) [inline testing/synergy/APKBUILD]: does unittests fail on alpine too or was those just copied from arch linux?

**Comment by clandmeter** [comment]
Closed due to inactivity.

**Pull request closed without merging**